packages/mgt-components/src/components/mgt-file-list/mgt-file-list.ts

```diff
@@ -542,12 +553,51 @@ export class MgtFileList extends MgtTemplatedTaskComponent implements CardSectio
    */
   protected renderFile(file: DriveItem): TemplateResult {
     const view = this.itemView;
-    return (
-      this.renderTemplate('file', { file }, file.id) ||
-      mgtHtml`
+    // if file is type SharedInsight, render Shared Insight File
+    if ((file as SharedInsight).lastShared) {
```

Comment:
If the goal is to type check file as SharedInsight, use a user-defined type gaurd here. Something like this:
```js
    ...
    // if file is type SharedInsight, render Shared Insight File
    if (isSharedInsight(file)) {
      return this.renderSharedInsightFile(file);
    }
    return (
      this.renderTemplate('file', { file }, file.id) ||
      mgtHtml`
      <mgt-file class="mgt-file-item" .fileDetails=${file} .view=${view}></mgt-file>
    `
    );
  }
	
   isSharedInsight = (file: Entity): file is SharedInsight => {
	   return 'lastShared' in file
	 }